.pre-commit-config.yaml

```
@@ -71,7 +71,7 @@ repos:
   #        args: ["--fix"]
 
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v3.1.0
+    rev: v4.0.0-alpha.8
```

```suggestion
    rev: v3.1.0
```
For the 34235434563456356th time, no alpha, ah you are just a bot.